test/testEnv.js

```diff
@@ -14,36 +14,77 @@
  * limitations under the License.
  */
 
-const senderPrivateKey = '0x'
-const payerPrivateKey = '0x'
+const dotenv = require('dotenv')
+
+let senderPrivateKey = '0x'
+
+dotenv.config()
 
 const auths = {
     nodeAPI: {
-        url: 'https://node-api.dev.klaytn.com/v1/klaytn',
+        url: 'https://node-api.klaytnapi.com',
```

Comment:
Since we define both dev and prod as env variables, do we need this?

Reply:
For now, we didn't try to set prod as env variables. 
If we set additional env variables for those, then sure we don't need this.